src/software/ai/navigator/path_planner/path_planner_performance_test.cpp

```diff
@@ -297,23 +297,18 @@ TEST_P(PlannerPerformanceTest, DISABLED_path_planner_performance)
     std::unique_ptr<PathPlanner> planner = std::get<0>(GetParam()).second();
     auto planner_test_case               = std::get<1>(GetParam());
 
-    auto start_time = std::chrono::high_resolution_clock::now();
+    auto start_time = std::chrono::system_clock::now();
     for (unsigned int i = 0; i < planner_test_case.num_iterations; i++)
     {
         planner->findPath(planner_test_case.start, planner_test_case.end,
                           planner_test_case.navigable_area, planner_test_case.obstacles);
     }
-    auto end_time = std::chrono::high_resolution_clock::now();
-
-    double duration_ms =
-        static_cast<double>(
-            std::chrono::duration_cast<std::chrono::nanoseconds>(end_time - start_time)
-                .count()) /
-        1000000.0;
+    double duration_ms = Time::millisecondsSince(start_time);
     double avg_ms = duration_ms / (static_cast<double>(planner_test_case.num_iterations));
 
     std::cout << std::endl << planner_test_case.name << ":" << std::endl;
 
+    // Performance was optimized in PR #1443
```

Comment:
Any particular reason this comment is here? Just not seeing how performance improvements are relevant?

Reply:
it's a benchmark for regressions, similar to the comment in cost_function_test.cpp